Analysis/Tasks/PWGHF/taskDDbarCorrelation.cxx

```diff
@@ -0,0 +1,395 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file taskDDbarCorrelation.cxx
+/// \brief D-Dbar analysis task - data-like, MC-reco and MC-kine analyses. For ULS and LS pairs
+///
+/// \author Fabio Colamaria <fabio.colamaria@ba.infn.it>, INFN Bari
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/HistogramRegistry.h"
+#include "AnalysisCore/HFSelectorCuts.h"
+#include "AnalysisDataModel/HFSecondaryVertex.h"
+#include "AnalysisDataModel/HFCandidateSelectionTables.h"
+
+using namespace o2;
+using namespace o2::framework;
+using namespace o2::aod::hf_cand_prong2;
+using namespace o2::analysis::hf_cuts_d0_topik;
+using namespace o2::framework::expressions;
+using namespace o2::constants::math;
+using namespace o2::aod::hf_ddbar_correlation;
+
+namespace o2::aod
+{
+using DDbarPairFull = soa::Join<aod::DDbarPair, aod::DDbarRecoInfo>;
+} // namespace o2::aod
+
+void customize(std::vector<o2::framework::ConfigParamSpec>& workflowOptions)
+{
+  ConfigParamSpec optionDoMCGen{"doMCGen", VariantType::Bool, false, {"Run MC-Gen dedicated tasks."}};
+  ConfigParamSpec optionDoMCRec{"doMCRec", VariantType::Bool, false, {"Run MC-Rec dedicated tasks."}};
+  workflowOptions.push_back(optionDoMCGen);
+  workflowOptions.push_back(optionDoMCRec);
+}
+
+#include "Framework/runDataProcessing.h"
+
+///
+/// Returns deltaPhi value in range [-pi/2., 3.*pi/2], typically used for correlation studies
+///
+double getDeltaPhi(double phiD, double phiDbar)
+{
+  return RecoDecay::constrainAngle(phiDbar - phiD, -o2::constants::math::PI / 2.);
+}
+
+///
+/// Returns deltaPhi value in range [-pi, pi], for resolution distributions
+///
+double getDeltaPhiForResolution(double phiD, double phiDbar)
+{
+  return RecoDecay::constrainAngle(phiDbar - phiD, -o2::constants::math::PI);
+}
```

Comment:
Where is this function used?

Reply:
It was used when I had an additional task for evaluating the Phi resolution. I removed that task, but had forgotten to remove this function